hiss/application/admin.py

```diff
@@ -1,8 +1,154 @@
+import datetime
+
+from django import forms
+from django.conf import settings
 from django.contrib import admin
+from django.db import transaction
+from django.utils import timezone
+
+from application.models import Application, Wave
+from user.models import User
+
+
+
+class ApplicationAdminForm(forms.ModelForm):
+    class Meta:
+        model = Application
+        fields = "__all__"
+        widgets = {
+            "gender": forms.RadioSelect,
+            "classification": forms.RadioSelect,
+            "grad_term": forms.RadioSelect,
+            "status": forms.RadioSelect,
+        }
+
+
+def create_rsvp_deadline(user: User, deadline: timezone.datetime) -> None:
```

Comment:
Please add a test for this function. In addition, please add a docstring indicating what this function is supposed to do.